src/components/Button/Button.jsx

```diff
@@ -17,7 +18,34 @@ export function Button({
   onClick,
   className,
   type,
+  to = false,
```

Comment:
if `to` is a string (👀  at propTypes) then is it valid to set the default value to `false`?